contracts/contracts/ccip/fee_quoter/messages.tolk

```diff
@@ -68,3 +68,9 @@ fun FeeQuoter_Costs.GetValidatedFee(): int {
         + SendExecutor_Costs.MessageValidationFailed()
     );
 }
+
+fun FeeQuoter_Costs.updatePrices() {
+    // when we support token transfers the message value amount will depend on the quantity of supported tokens
+    return ton("0.002");
```

Comment:
Should this be a configuration or if we need to change this value, will we perform a contract upgrade?